src/category_theory/sites/plus.lean

```diff
@@ -337,4 +347,8 @@ begin
   rw [← category.assoc, ← J.to_plus_naturality, category.assoc, J.to_plus_plus_lift],
 end
 
+instance plus_functor_preserves_zero_morphisms [preadditive D] :
```

Comment:
What's the autogenerated name? (Same for below).

Reply:
If I remembered correctly, it just said something like “fails to generate names” for sheafification preserving zero morphisms, after that I just manually wrote down the names